packages/babel-plugin-syntax-import-module-attributes/README.md

```diff
@@ -1,6 +1,6 @@
 # @babel/plugin-syntax-import-module-attributes
```

Comment:
```suggestion
# @babel/plugin-syntax-module-attributes
```

Since the proposal is just called "module attributes"

Reply:
Made the changes 👍